addons/payment_custom/__manifest__.py

```diff
@@ -1,10 +1,11 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 {
-    'name': 'Custom Payment Modes',
+    'name': 'Payment Provider: Custom Payment Modes',
     'version': '2.0',
     'category': 'Accounting/Payment Providers',
-    'summary': 'Payment Provider: Custom payment modes',
+    'sequence': 350,
+    'summary': "Base of custom payment methods as Wire Transfer.",
```

Comment:
Maybe we can make a summary more like the one of `demo`

```suggestion
    'summary': "A generic payment provider for custom flows like Wire Transfer or On-Site Payments",
```